pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/BrokerHostUsage.java

```diff
@@ -32,15 +41,26 @@
 public class BrokerHostUsage {
```

Comment:
Sounds good to me. Even some of the OS-indipendent methods (eg: used mem, processors count) could be put in a base abstract class.